### DISCUSSION TITLE
Allow users to choose other testing frameworks for their component and e2e tests

### DISCUSSION THREAD

**Discussion by christian-bromann** [remix-run/remix#5870] [Proposals]
Hey there 👋 

When starting a Remix project using either a Grunge, Blues or Indie stack users will always get Cypress as testing framework. Cypress is a great tool and many developers love it but there are also others that people might want to consider. I am maintaining the WebdriverIO project which has great support for UI/component and e2e tests of React applications.

I wonder if the Remix team would be open to make the choice of testing frameworks optional, I am happy to contribute to all necessary places.

**Comment by sergiodxa** [comment]
Stacks are intended to be opinionated about the tech choices, it’s possible to fork them to change any part of them and use them when creating a new project, the official stacks are not the only available stacks.
